### PR TITLE
Bug fixes for display_script branch

### PR DESCRIPTION
**Pull request by mattjmeier**

### CODE REVIEW

**Comment by cookkate** [comment]
Yeah please do that. I knew I was going to end up generating a conflict on your end, sorry.  I will check this one out!

File: .github/workflows/tests.yml [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+          use-only-tar-bz2: true # This needs to be set for caching to work properly, according to others
+      # - name: Set up R ${{ matrix.r-version }}
+      #   uses: r-lib/actions/setup-r@v2
+      #   with:
+      #     r-version: ${{ matrix.r-version }}
+      - name: Install extra dependencies
+        run: |
+          R -e "chooseCRANmirror(1, graphics=FALSE); \
+          remotes::install_github('bwlewis/crosstool'); \
+          install.packages('cellWise')"
```
Comment (cookkate) [inline .github/workflows/tests.yml:54]: I assume this one is just annoying and had to be installed this way

**Comment by cookkate** [comment]
Cool! I made a couple of tweaks, seems to run well. One question: do we want to check that the conda install does anything? Or is it assumed that someone will be looking at the 'conda list' output. Or is this just backup in case things go wrong.

**Comment by mattjmeier** [comment]
Glad it works for you too! The conda list was for troubleshooting when I was trying to figure everything out. But I think if it crashes and something doesn't work right then it will fail the tests too which is kind of helpful in it's own way.

**Pull request closed without merging**